README.md

```diff
@@ -6,39 +6,6 @@ The [notaryproject.dev][] website, built using [Hugo][] and hosted on [Netlify][
 
 Visit [https://gitpod.io/#https://github.com/notaryproject/notaryproject.dev](https://gitpod.io/#https://github.com/notaryproject/notaryproject.dev) to launch a [Gitpod.io](https://gitpod.io) IDE that will allow you to build, preview and make changes to this repo.
 
```

Comment:
Moving the following steps into a separate CONTRIBUTING.md is good. Do we need to add a link to CONTRIBUTING.md here?

Reply:
It will be linked on the right side of the repo, but i'll add link here too